README.md

```diff
@@ -506,9 +506,9 @@ opt_in_rules: # some rules are turned off by default, so you need to opt-in
 analyzer_rules: # Rules run by `swiftlint analyze`
   - explicit_self
 
-included: # paths to include during linting. `--path` is ignored if present.
+included: # case-sensitive paths to include during linting. `--path` is ignored if present.
```

Comment:
Since it ends with a period, the first letter should be uppercase I think:

```suggestion
included: # Case-sensitive paths to include during linting. `--path` is ignored if present.
```

Reply:
The majority of the inline comments start with lowercase letters and don't end in a full stop, so I thought to go the other way, and change some other exceptions to get them inline (pun not intended). What do you think?